Parent/Parent/ParentAppDelegate.swift

```diff
@@ -296,3 +296,20 @@ extension ParentAppDelegate: UNUserNotificationCenterDelegate {
         completionHandler([.alert, .sound])
     }
 }
+
+extension ParentAppDelegate {
+    func application(_ application: UIApplication, continue userActivity: NSUserActivity, restorationHandler: @escaping ([UIUserActivityRestoring]?) -> Void) -> Bool {
+        if userActivity.activityType == NSUserActivityTypeBrowsingWeb, let url = userActivity.webpageURL, let login = GetSSOLogin(url: url) {
```

Comment:
I think we should use the parent specific code right away so that it doesn't break later on down the line when parent gets its own client_id and we forget about this.

Reply:
Scratch this idea.